SpriterDotNet.UnitTests/ForParsing/ForSpriterParser.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SpriterDotNet.Parser;
+
+namespace SpriterDotNet.UnitTests.ForParsing
+{
+    [TestFixture]
+    public class ForSpriterParser
+    {
+        [Test]
+        public void WhenInstantiated_HasAnXmlParserInItsParsers()
+        {
+            var parsers = SpriterParser.Parsers;
+            parsers.Count.Should().Be(1, "there is only one parser by default");
```

Comment:
I think it would be good to check it there's at least one parser and if any of the parsers are xml, not necessarily the first one.